### DISCUSSION TITLE
1주차 스터디 회고

### DISCUSSION THREAD

**Discussion by alanhakhyeonsong** [Next-Level-Study/effective-java-study#3] [스터디 회고]
한 줄 회고와 건의사항을 말씀해주세요:)

**Comment by alanhakhyeonsong** [comment]
다들 열심히 준비해주시고 참여해주셔서 유익한 스터디가 됐습니다. 작심 3일이 아니라 꾸준하게 스터디 완료가 될 수 있도록 최선을 다해 리딩하겠습니다!

**Comment by opgumo3** [comment]
다함께 공부하니 의욕 뿜뿜💥 
더 열심히 자료 조사하고 정리해야겠습니다.
마지막까지 다들 포기하지 말고 열심히 해봐요 💚

**Comment by sdh46** [comment]
혼자 공부할 때 모르는 지도 모르고 넘어간 부분을 같이 한번 짚고 넘어갈 수 있어서 더 많이 얻어갈 수 있어서 좋았습니다.

**Comment by Kurmarkpzgr** [comment]
확실히 혼자 공부할 때 보다 많은 분들과 의견 공유를 하니까
다양한 관점에서 한 번 더 생각해 볼 수 있었어서 좋았습니다.

**Comment by JeongJaew0n** [comment]
스터디 처음했는데 다들 열심히 하셔서 동기부여가 더욱 더 되고, 한 주를 뿌듯하게 보낼 수 있었습니다!

**Comment by narangd0** [comment]
굿굿! 준비가 미흡했지만 다른 분들 본받아 앞으로 더 노력하겠습니다!
각 주에 미해결된 내용은 좀 더 조사해서 차주 발표 내용에 추가해도 좋을 것 같습니다. (복습도 되고 좋을 듯)
- 플라이웨이트 패턴
- 싱글톤 패턴에서 public 필드, private 필드 활용 방법의 차이점
